packages/frontend/src/build/buildMetaImages.ts

```diff
@@ -4,12 +4,18 @@ import { mkdirpSync } from 'fs-extra'
 import { Server } from 'http'
 import puppeteer from 'puppeteer'
 
+import { getConfig } from './config'
+
 main().catch((e) => {
   console.error(e)
   process.exit(1)
 })
 
 async function main() {
+  const env = process.env.DEPLOYMENT_ENV ?? 'production'
```

Comment:
we already have this in the buildPages, do we need it here too?

Reply:
`buildPages` runs in a separate process, so yes, I think we need it here too.